src/app/shared/object-list/my-dspace-result-list-element/item-list-preview/themed-item-list-preview.component.html

```diff
@@ -0,0 +1,16 @@
+<div>
+    <div class="row" *ngIf="collection">
+        <div class="col-12">
+            <div class="col-12">
+                <i>
+                    <small>{{"collection.listelement.badge" | translate}}:</small>
+                </i>
+                &nbsp;
+                <a [href]="uiBaseUrl+'/collections/'+collection.uuid">
```

Comment:
Instead of hardcoding the `href`, Angular recommends using `[routerLink]` as that takes into account the base URL automatically.  So, I think this can be replaced with something like:
`<a [routerLink]="['/collections', collection.uuid]">`
(I haven't tested that exact syntax, but we have a ton of usages of `[routerLink]` all over our codebase, so there's a lot of examples)